bot/exts/help_channels/_cog.py

```diff
@@ -642,3 +644,16 @@ async def helpdm_command(
         else:
             await _caches.help_dm.delete(ctx.author.id)
         await ctx.send(f"{constants.Emojis.ok_hand} {ctx.author.mention} Help DMs {state_str}!")
+
+    @tasks.loop(hours=1)
+    async def check_channel_consistency(self) -> None:
+        """
+        Check number of available help channels periodically.
+
+        If number of available help channels is incorrect, reload cog.
+        """
+        log.trace("Checking channel consistency")
+        available_help_channels = set(_channel.get_category_channels(self.available_category))
+        if len(available_help_channels) != 3:
```

Comment:
the maximum amount of available channels is configurable in the config, this should use the config value

Reply:
Fixed.